RileyLinkKit/PumpOpsSession.swift

```diff
@@ -65,7 +65,7 @@ extension PumpOpsSession {
 
         let shortPowerMessage = PumpMessage(settings: settings, type: .powerOn)
 
-        if pump.pumpModel == nil || !pump.pumpModel!.hasMySentry {
+        if pump.pumpModel == nil || !(pump.pumpModel!.hasMySentry && pump.useMySentry) {
```

Comment:
I'm not 100% sure, but I think the wakeup requirements vary based on the model number of the pump, and not whether mysentry is enabled, so I think this might be adding a slower wakeup for pumps that do not need it.

Reply:
Reverted.